UPGRADE.md

```diff
@@ -22,12 +22,27 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 - added [Microservice Product Search Export](https://github.com/shopsys/microservice-product-search-export)
     - check changes in the `docker-compose.yml` template you used and replicate them, there is a new container `microservice-product-search-export`
     - `parameters.yml.dist` contains new parameter `microservice_product_search_export_url`
+- to use custom postgres configuration check changes in the `docker-compose.yml` templates and replicate them, there is a new volume for `postgres` container
+    - PR [Improve Postgres configuration to improve performance](https://github.com/shopsys/shopsys/pull/444)
+    - Move data from `project-base/var/postgres-data` into `project-base/var/postgres-data/pgdata`. Instructions for Ubuntu:

```

Comment:
I love those instructions for Ubuntu :+1: :slightly_smiling_face: But for all the other platforms it would be nice to be more descriptive - I would rewrite it in this spirit: "Stop all the running containers and move data from `project-base/var/postgres-data` into `project-base/var/postgres-data/pgdata`. The folder must have the correct permissions (i.e. ???). After that, you can start your containers once again. To provide you with a better image of what exactly needs to be done, there are instructions for Ubuntu:...". 

Does it make sense to you?

Reply:
better?

Reply:
awesome! :+1: